docs/keto/engines/acp-ory.md

```diff
@@ -547,25 +526,23 @@ developed over the years at ORY.
 
 ### URNs
 
-> “There are only two hard things in Computer Science: cache invalidation and
-> naming things.” -- Phil Karlton
+> “There are only two hard things in Computer Science: cache invalidation,
+> naming things, and off-by-one errors.” -- Phil Karlton
```

Comment:
Added this joke, not sure if it is from Phil Karlton. But I think it adds a little breath of fresh air while reading through this long piece of security documentation.

Reply:
:)